Documentation/network/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -90,9 +90,9 @@ Install cluster one
     .. code-block:: bash
 
         cilium install \
-            --azure-resource-group "${AZURE_RESOURCE_GROUP}" \
-            --cluster-id 1 \
-            --config "cluster-pool-ipv4-cidr=10.10.0.0/16"
+            --set azure.resourceGroup="${AZURE_RESOURCE_GROUP}" \
+            --set cluster.id=1 \
+            --set ipam.operator.clusterPoolIPv4PodCIDRList={"10.10.0.0/16"}
```

Comment:
```suggestion
            --set ipam.operator.clusterPoolIPv4PodCIDRList='{10.10.0.0/16}'
```

Seems to work better to me (especially in case multiple CIDRs are specified). For instance, `...CIDRList='{10.10.0.0/16,10.20.0.0/16}'` leads to 

```yaml
ipam:
  mode: kubernetes
  operator:
    clusterPoolIPv4PodCIDRList:
    - 10.10.0.0/16
    - 10.20.0.0/16
```

The same applies also to the one below.